account_check_printing_report_base/models/res_config_settings.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Iterativo - Manuel Marquez <buzondemam@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    check_layout_verification = fields.Selection([
```

Comment:
Don't add this option, as it's simpler: if you put a layout in the journal, then use that one. If not, use the company's one.